### DISCUSSION TITLE
Forcing Language to be X  if language not found in a list

### DISCUSSION THREAD

**Discussion by ngcheeyuan** [SYSTRAN/faster-whisper#890] [Q&A]
Hi, as per title. I am trying to restrict the possible range of languages predicted. May I know if it is possible to do so with faster-whisper?

**Comment by trungkienbkhn** [comment]
Not yet, FW currently does not support this.

**Reply by ngcheeyuan** [reply]
@trungkienbkhn got it. Thanks.
